### ISSUE TITLE
дз5

### ISSUE THREAD

**Issue by ibragimova765** [kalininei/CFDCourse24#7]
Добрый день. При попытке собрать решение в файле cavern_2d_simple_test.cpp возникает ошибка ошибка в следующих местах 
![image](https://github.com/kalininei/CFDCourse24/assets/34399224/1b5ad175-badc-4ee3-ba75-1837dc7e5f7d)
В чем может быть проблема?

**Comment by kalininei** [comment]
Не знаю. Может вы старый файл перетащили в новый коммит. Или наоборот новый файл в старый коммит.
Я там в какой-то момент переименовал `TimeDependentWriter` в `TimeSeriesWriter`. Но во все файлы изменения внёс.

Посмотрите `src/cfd24/grid.vtk.hpp`. Там как этот класс называется? Если `TimeSeriesWriter`, значит это новая версия.
И нужно использовать это название вместо подчёркнутого.
Если же там `TimeDependentWriter`, то вы, выроятно не обновили проект после переключения веток. Для этого надо ZERO_CHECK собрать.

**Comment by ibragimova765** [comment]
Да, помогло, спасибо. У меня еще в cavern_2d_fvm_simple_test.cpp возникает ошибка следующего вида
![image](https://github.com/kalininei/CFDCourse24/assets/34399224/a094d2cc-cecd-4842-bc8f-ec14ba536e5e) А здесь в чем проблема?

**Comment by kalininei** [comment]
Вы что, пытаетесь перетаскивать файлы из старых коммитов в последнюю версию проекта? Зачем?
В последней версии у меня нет функции `assemble_fvm_faces_dudn`. У меня там класс `FvmFacesDn`.
Если вы делаете работу на основе новой версии, то надо использовать этот класс для вычисления производной.
Он у меня там и использовался в этом файле.
